changelog.md

```diff
@@ -1,3 +1,6 @@
+# 3.17.4
+- [ADDED] Support silent: true in bulk update [#5200](https://github.com/sequelize/sequelize/issues/5200)
+
```

Comment:
Don't add your own version, just put feature.
A feature addition should be a minor bump (not a patch) in any case, but it's best to just leave it up to us depending on what and when release (also does not make people think there's a new release by looking at the changelog :))